highfidelity/hifi
# Updating keyboard mapping .jpg to reflect keyboard/mouse mapping

- Fix for MS[#14555](https://highfidelity.manuscript.com/f/cases/14555/help-doesn-t-match-fly-down-behavior)

**Test Plan**
- Launch Interface
- Open `Menu > Help > Controls Reference`
- Switch to `Keyboard + Mouse` tab using arrow buttons in pop-up window
- "Fly down" mapping should be shown as `C`
